src/components/FeedbackWidget.tsx

```diff
@@ -140,7 +140,7 @@ const FeedbackWidget: React.FC<FeedbackWidgetProps> = ({ location = "" }) => {
 
   useKeyPress(`Escape`, handleClose)
 
-  if (!location.includes("/en/")) return null
```

Comment:
`location` on server-side is not going to give us the same url. Since we are only interested in getting the current `locale`, we can get it from the `intl` context.